### DISCUSSION TITLE
C# Materials

### DISCUSSION THREAD

**Discussion by thechandanbhagat** [Training-by-Chandan/solid-csharp-journey#11] [General]
[Fundamentals of Computer Programming with C#](https://github.com/Training-by-Chandan/super-duper-3pm/files/8357362/Fundamentals-of-Computer-Programming-with-CSharp-Nakov-eBook-v2013.pdf)

[C# Bible](https://github.com/Training-by-Chandan/super-duper-3pm/files/8357363/C.Bible.pdf)
Link Here

**Comment by Balkrishna30** [comment]
give access to Balkrishna

**Reply by thechandanbhagat** [reply]
@Balkrishna30 check the email invite
